### PR TITLE
Code reviews diya

### PR DESCRIPTION
**Pull request by diyarenny**
This PR contains the following points:
- Reviewers guidelines on code reviews
- Examples of code reviews

Please review my PR and give suggestions.

### CODE REVIEW

**Review by JouharaAlwardi — APPROVED** [review]
The PR looks good.

**Review by oisinallen99 — APPROVED** [review]
looks good for me

File: README.md [hunk 1]
```diff
@@ -18,6 +18,64 @@ from getting into your project at an early stage of the software development pro
...
+
+### 4. Reviewers guidelines on code reviews
+
+---
+
+A code review is a process where someone other than the engineers of a piece of code examines that code. The following table showcases what code reviewers Look For in a code review:
+
+&nbsp;
+
+| Features  | Explaination |
```
Comment (oisinallen99) [inline README.md]: typo, should be Explanation

Comment (diyarenny) [reply README.md]: Fixed it

**Comment by xtreger** [comment]
Looks perfect, approving PR.

**Review by xtreger — APPROVED** [review]

**Pull request merged**